src/com/massivecraft/massivetickets/cmd/CmdTicketsYield.java

```diff
@@ -30,7 +30,7 @@
 	public CmdTicketsYield()
 	{
 		// Parameters
-		this.addParameter(TypeMPlayer.getOnline(), "player");
+		this.addParameter(TypeMPlayer.getAny(), "player");

```

Comment:
I think it's a good idea to be able to yield tickets even if the player is offline. Stupid we did not support that before.

I see however that you have not made this change for `CmdTicketsDone` could the other pull request be invalidated simply by allowing `TypeMPlayer.getAny()` for `CmdTicketsDone`?

Reply:
Absolutely. My initial concern was that this would still count towards points and high scores when closing Ghost Tickets. Reconsidering, however, this is a non-issue if it allows the Ghost Tickets to be closed as soon as noticed without sitting for months.